Kconfig

```diff
@@ -310,7 +302,7 @@ menu "LVGL configuration"
 					Error diffusion dithering gets a much better visual result, but implies more CPU consumption and memory when drawing.
 					The increase in memory consumption is (24 bits * object's width)
 
-			config LV_DISP_ROT_MAX_BUF
+			config LV_DISPLAY_ROT_MAX_BUF
```

Comment:
This config is not used anymore. We have a [helper function](https://github.com/lvgl/lvgl/blob/master/src/draw/sw/lv_draw_sw.h#L193) and user can allocate memory as he wants.

Reply:
fixed: https://github.com/lvgl/lvgl/pull/5239

Reply:
Thanks, merged.